.github/workflows/ci.yml

```diff
@@ -81,7 +81,7 @@ jobs:
           php-version: '8.0'
           tools: composer:v2
           coverage: none
-          extensions: decimal
+          extensions: :pdo
```

Comment:
Is there a way to tell what extensions are loaded on the CI build? It looks like I need to disable soap as well and add a stub for `SoapFault`, but what others?